controller/space_areas.go

```diff
@@ -46,7 +47,7 @@ func (c *SpaceAreasController) List(ctx *app.ListSpaceAreasContext) error {
 			return jsonapi.JSONErrorResponse(ctx, err)
 		}
 		for _, a := range areas {
-			logrus.Info("Found space area with ID=", a.ID.String())
+			log.Info(ctx, map[string]interface{}{"area_id": a.ID}, "Found space area with id %s", a.ID)
```

Comment:
generally, yes. that is the purpose. But I am not going to change all the error messages :) that add one or to values on them.